packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.base.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { IVerticalDividerProps, IVerticalDividerPropsStyles, IVerticalDividerStyles } from './VerticalDivider.types';
+import { classNamesFunction } from '../../Utilities';
+const getClassNames = classNamesFunction<IVerticalDividerPropsStyles, IVerticalDividerStyles>();
+
+export const VerticalDividerBase = (props: IVerticalDividerProps) => {
+  const { styles } = props;
+  const classNames = getClassNames(styles, { theme: props.theme, getClassNames: props.getClassNames });
```

Comment:
nit: include the `theme` and the `getClassNames` in the destructure above same as you did with styles